docs/source/faq.rst

```diff
@@ -0,0 +1,154 @@
+``datanucleus.cache.level2.type`` property to ``none``. There may be slight performance drop per server in result of
+turning off level2 cache. Level1 cache should be kept as it works within one transaction, therefore not affecting
+cluster deployment.
+
+More information about the DataNucleus can be found on it project page (link_). There is also available caching doc_.
+
+.. _link: http://www.datanucleus.org/
+.. _doc: http://www.datanucleus.org/products/accessplatform_4_0/jpa/cache.html
+
+How to use SSL for database connections in MOTECH?
+--------------------------------------------------
+First generate and install your SSL certificates. After this is done you will need to configure your database to use SSL.
+For MySQL you can follow instructions_ from official MySQL page.
+Next step is to point JVM variables to your SSL certificates. Look here_ for more information about these variables.
+
+In order to establish a secure SSL connection the JDBC URL needs to be updated with the JDBC driver property ``'useSSL'=true'``
+eg. if you are using mySQL at localhost the JDBC URL should be ``jdbc:mysql://localhost:3306/?useSSL=true``. As the URL
+entered on the MOTECH bootstrap page is only the base for actual database connections urls, you should provide URL with
+SSL parameter to appropriate config files.
+
+For MDS it can be set by editing ``~/.motech/config/datanucleus.properties`` and changing the property ``javax.jdo.option.ConnectionURL``.
+For quartz you have to edit the variable ``org.quartz.dataSource.motechDS.URL`` in ``quartz.properties``. If your config
+source is FILE, then you have to edit the file in scheduler directory in ``~/.motech/config``. If your source is UI, you
+have to change the quartz configuration through the manage modules section in the Admin UI.
+
+.. _instructions: http://dev.mysql.com/doc/refman/5.6/en/using-ssl-connections.html
+.. _here: https://docs.oracle.com/cd/E29585_01/PlatformServices.61x/security/src/csec_ssl_jsp_start_server.html
+
+MOTECH is not working as expected, how can I check what's wrong?
+----------------------------------------------------------------
+All the problems that MOTECH encounters are logged. The logs can be checked in two ways. If MOTECH is working you can
+access logs by choosing ``Server Log`` from ``Admin`` panel on the UI. There you can also specify logging level for
+specific packages - this is done by selecting wanted log level in ``Server Log -> Log options`` panel.
+
+If access to UI is not possible, for standard MOTECH installations on Tomcat logs can be also found in ``catalina.out``
+file located in the logs folder in the Tomcat installation. For example for MOTECH deployed on Tomcat 7.0.62 the ``catalina.out``
+is located in ``~/apache-tomcat-7.0.62/logs`` folder. The content of the log file can be displayed by any text editor
+or in unix system terminal by for example ``less catalina.out`` command.
```

Comment:
why not in a Windows terminal? 

I know this is nitpicking, but really, we can drop such details like using less, more or whatever, if they don't know how to read a file, a faq won't help them anyway.